### ISSUE TITLE
Building packages is completely broken

### ISSUE THREAD

**Issue by flamusdiu** [hawaii-desktop/hawaii-pkgbuilds#8]
I have been trying to build hawaii for the last few days. Looks like all of the KDE Framework stuff is missing from the repo making the hawaii-shell not build at all. Also, the versions are still messed up (as noted in #5). Lastly, a number of PKGBUILDs have "master" for the git branch but they are suppose to be "stable" instead.

**Comment by flamusdiu** [comment]
@plfiorini =D I was doing a lot of that but seems plenty of things have changed so I wasn't sure exactly where you were going with it.

**Comment by plfiorini** [comment]
Packages to get the shell running:

* greenisland-git
* hawaii-shell-git
* hawaii-icon-themes-git

Some repos were merged with hawaii-shell to simplify development and packaging.

AUR was updated a few minutes ago, but I haven't tested the build yet.
Unfortunately there are a few KDE Frameworks dependencies from git, the most importand being libkscreen-git. Once released (Jan 22) the packages can point to the non-git versions.

**Comment by flamusdiu** [comment]
I will to look at it again. Thanks for the update!

**Comment by plfiorini** [comment]
Packages updated, build tested on VMware.
Builds and run (in nested mode, with weston).

**Comment by flamusdiu** [comment]
@plfiorini I'll have to check it out this weekend.

**Comment by flamusdiu** [comment]
@plfiorini this is still failing for me.

When I do the chroot build, it doesn't create the "repo" directory under ~/chroot/root which I have to do manually and make sure my normal user account can write to it.

Also, I changed the top of the `build-hawaii` script starting at Line 12 to this:

```
    if (grep -q "any" ${pkg}/PKGBUILD); then
		file=${pkg}-*-1-any.pkg.tar.xz
	else
		file=${pkg}-*-1-x86_64.pkg.tar.xz
	fi
```

Some packages have `x86_64` while others build as `any` . Also I removed the 'v' from it as well. The version numbers don't have 'v' at the beginning of them any more.

When building `hawaii-icon-themes-git,` it fails with the following error:
```
==> Making package: hawaii-icon-themes-git 20150127.9f74bf2-1 (Sun Feb 15 16:59:42 UTC 2015)
==> Checking runtime dependencies...
==> Installing missing dependencies...
error: target not found: captiva-icons-git
error: target not found: faba-icon-theme-git
==> ERROR: 'pacman' failed to install missing dependencies.
```
I guess they are the following packages from AUR:
  - https://aur.archlinux.org/packages/captiva-icons-git/
  - https://aur.archlinux.org/packages/faba-icon-theme-git/
